syz-fuzzer/testing.go

```diff
@@ -148,7 +149,39 @@ func checkMachine(args *checkArgs) (*rpctype.CheckArgs, error) {
 	if err := checkSimpleProgram(args, features); err != nil {
 		return nil, err
 	}
-	return checkCalls(args, features)
+	res, err := checkCalls(args, features)
+	if err != nil {
+		return nil, err
+	}
+	log.Logf(0, "checking modules...")
```

Comment:
Please factor this into a separate function and move this to pkg/host, this code should be OS-specific.
Please also add unit test for parsing of file contents.

Reply:
Moved to pkg/host